lib/rpcs/v2_0_0/get_entity.rs

```diff
@@ -21,22 +22,43 @@ pub enum EntityIdentifier {
     EntityAddr(EntityAddr),
 }
 
+/// An addressable entity with named keys and entry points.
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Clone)]
+pub struct AddressableEntity {
+    pub entity: CasperTypesAddressableEntity,
+    pub named_keys: NamedKeys,
+    pub entry_points: Vec<EntryPointValue>,
+}
+
 /// An addressable entity or a legacy account.
-#[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Clone)]
 pub enum EntityOrAccount {
     /// An addressable entity with named keys and entry points.
-    AddressableEntity {
-        /// The addressable entity.
-        entity: AddressableEntity,
-        /// The named keys of the addressable entity.
-        named_keys: NamedKeys,
-        /// The entry points of the addressable entity.
-        entry_points: Vec<EntryPointValue>,
-    },
+    AddressableEntity(AddressableEntity),
     /// A legacy account.
     LegacyAccount(Account),
 }
 
+impl EntityOrAccount {
+    /// Returns the addressable entity if present.
+    pub fn addressable_entity(&self) -> Option<AddressableEntity> {
```

Comment:
I'd probably change this to return a reference and let the user decide if they want to clone or not
```suggestion
    pub fn addressable_entity(&self) -> Option<&AddressableEntity> {
```

Reply:
thank you very much good point